openedx/features/offline_mode/utils.py

```diff
@@ -36,6 +37,8 @@ def generate_offline_content(xblock, html_data):
     try:
         save_xblock_html(tmp_dir, xblock, html_data)
         create_zip_file(tmp_dir, base_path, f'{xblock.location.block_id}.zip')
+    except Http404:
+        log.error(f'Block {xblock.location.block_id} for course {xblock.location.course_key} not found.')
```

Comment:
I think log message should be enhanced by stating for example that "block X cannot be fetched from course Y during offline content generation."

Reply:
Done